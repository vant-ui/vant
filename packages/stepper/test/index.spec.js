import Stepper from '../';
import demoTest from '../../../test/demo-test';

demoTest(Stepper);
