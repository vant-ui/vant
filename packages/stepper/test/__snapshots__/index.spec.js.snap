// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders stepper correctly 1`] = `
<section class="van-doc-demo-section demo-stepper" style="background:undefined;">
  <section class="van-doc-demo-block">
    <h2 class="van-doc-demo-block__title">基础用法</h2>
    <div class="van-stepper">
      <button class="van-stepper__minus van-stepper__minus--disabled"></button>
      <input type="number" value="1" class="van-stepper__input">
      <button class="van-stepper__plus"></button>
    </div>
  </section>
  <section class="van-doc-demo-block">
    <h2 class="van-doc-demo-block__title">禁用状态</h2>
    <div class="van-stepper">
      <button class="van-stepper__minus van-stepper__minus--disabled"></button>
      <input type="number" disabled="disabled" value="1" class="van-stepper__input">
      <button class="van-stepper__plus van-stepper__plus--disabled"></button>
    </div>
  </section>
  <section class="van-doc-demo-block">
    <h2 class="van-doc-demo-block__title">高级用法</h2>
    <div class="van-stepper">
      <button class="van-stepper__minus"></button>
      <input type="number" value="9" class="van-stepper__input">
      <button class="van-stepper__plus"></button>
    </div>
  </section>
</section>
`;
